code/modules/research/designs/mining_designs.dm

```diff
@@ -131,3 +131,13 @@
 	build_path = /obj/item/borg/upgrade/modkit/aoe/turfs
 	category = list("Mining Designs", "Cyborg Upgrade Modules")
 	departmental_flags = DEPARTMENTAL_FLAG_CARGO
+	
+/datum/design/miningsatchel_holding

```

Comment:
you can also modularize this by making a respective file in the yogs folder and all.